.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: scala
+sudo: false
+# apparently the hosts file needs to be prefilled
```

Comment:
is this still necessary? the issue linked to is a couple years old and is marked as closed

Reply:
I'm not fluent in Travis, took it from another project.